### PR TITLE
Cleaned up our wiki

### LINKED ISSUE #104: Clean up & Merge wiki-pages
As developers, we want to clean up our wiki (merge individual wiki pages, refresh home page, etc.) to give the team as well as external viewers a better overview.

---

## :white_check_mark: &nbsp; Conditions of satisfaction

- [ ] There should be no more duplicate information in the wiki.
- [ ] It was checked if some pages can be merged.
- [ ] There should be a visually attractive start page into the wiki, all links have to work.
- [ ] The sidebar of the wiki should be tidy.
- [ ] Outdated content of individual pages (e.g. Homework-ToDos) has been deleted.
- [ ] Make the key aspects of all pages more visible in the Wiki.

### PR DESCRIPTION
**Pull request by SanJSp**
Fixes #104 

## Changes
I changed multiple things in the wiki:
- fixed typos
- translated ssot documentation
- renamed files for a clearer structure given only their filenames
- merged vision, architecture and user interaction into one file
- merged ssot-deutsch and ssot
- Home is now easier to navigate

*describe problem here*
Wiki had duplicates and information scattered all around

## Screenshots
*If the frontend has been changed, please add a screenshot of before and after the changes have been made*

## Checklist before merge

**Developer's responsibilities**
* [ ] **Assign** one or two developers
* [ ] **Change code** if reviewer(s) has/have requested it
* [ ] **Pull request build** has passed
* [ ] **tested locally** (in at least chrome & firefox if frontend)
* [ ] updated the **documentation**
* [ ] added **tests** where necessary

### CODE REVIEW

File: wiki/Documentation-single-source-of-truth.md [hunk 1]
```diff
@@ -119,22 +127,22 @@ Hierbei ändert sich nichts im Vergleich zu Daniels Aufschlag einer SSoT. Nutzer
...
-- id :heavy_check_mark: 
-- Label -> name :heavy_check_mark: 
-- Vorgänger :heavy_check_mark: 
-- Nachfolger :heavy_check_mark: 
-- ~~Position~~ -> berechnet :heavy_check_mark: 
-- ~~Größe~~ -> `config.json` :heavy_check_mark: 
-- _RPA-Task & Application & Parameters_ :heavy_check_mark: 
+**BPMN requires:**
+- id :heavy_check_mark:
+- Label -> name :heavy_check_mark:
```
Comment (WolfgangDaniel) [inline wiki/Documentation-single-source-of-truth.md:132]: We should decide for all the bullet points (this also goes for all the following bullet points) whether each bullet point starts with a capital letter or not. It should be consistent though.

File: wiki/How-to-style-using-CSS.md [hunk 2]
```diff
@@ -304,16 +304,94 @@ function SpaceDemo() {
...
+
+### Conventions
+<details><summary>We agreed on
+</summary><p>
 
+- naming
+  For the naming of classes and ids please use **hyphens** consistently.
+  For example, don't call the class `buttonBackground` and instead call it `button-background`.
+- sizing
+  Try to use only relative units (vw,vh,rem,%) to size elements and **not** absolut units (px)
```
Comment (WolfgangDaniel) [inline wiki/How-to-style-using-CSS.md:389]: Behind naming and sizing we should put a colon

File: wiki/How-to-write-tests.md [hunk 3]
```diff
@@ -64,16 +64,16 @@ You can also use [this short Jest cheatsheet](https://devhints.io/jest)
...
 
 ### What to test?
-#### Do 
+#### Do
 * Utility methods
 * Complex implementations (eg: algorithms)
 * Anything that has edge cases
 * Core business logic
-* Services that are high-risk
+* high-risk Services
```
Comment (WolfgangDaniel) [inline wiki/How-to-write-tests.md:76]: Same here again. For each bullet point the first letter should be a capital letter or not

File: wiki/Team-Scrum-process.md [hunk 4]
```diff
@@ -1,42 +1,42 @@
...
 
-### General 
+### General
 - One sprint is usually three weeks long.
-- The teams core work time is monday (9am-3pm) and wednesday (9am-3pm)
+- The teams' core work time is Monday (9 am-3 pm) and Wednesday (9 am-3 pm)
 - Each member of the team works about 16 hours per week
 
-Update: Starting March 8, the team will work on the project four days a week. Monday-Wednesday will be the core work days that each team member will work. The fourth workday can be any of the days from Thursday through Sunday. 
+Update: Starting March 8, the team will work on the project four days a week. Monday-Wednesday will be the core workdays that each team member will work. The fourth workday can be any of the days from Thursday through Sunday.
```
Comment (WolfgangDaniel) [inline wiki/Team-Scrum-process.md:8]: We can update this now as we are starting with full time work next monday

Comment (WolfgangDaniel) [reply wiki/Team-Scrum-process.md:8]: Same goes for the daily scrum

**Review by WolfgangDaniel — CHANGES_REQUESTED** [review]

File: wiki/Team-Scrum-process.md (hunk 4 above)
Comment (kej-jay) [reply wiki/Team-Scrum-process.md:8]: +1

**Comment by kej-jay** [comment]
I would appreciate a [custom sidebar](https://docs.github.com/en/github/building-a-strong-community/creating-a-footer-or-sidebar-for-your-wiki#creating-a-sidebar) so we could order and group pages based on their content and topic.

File: wiki/Team-Scrum-process.md [hunk 5]
```diff
@@ -1,42 +1,40 @@
...
-
-Update: Starting March 8, the team will work on the project four days a week. Monday-Wednesday will be the core work days that each team member will work. The fourth workday can be any of the days from Thursday through Sunday. 
+- The teams' core work time is Monday-Wednesday (9 am-3 pm) and a fourth flexible day that each team member can set himself
+- Each member of the team works about 32 hours per week
 
 ### Meetings
 
 **1. Daily Scrum:**
-Limited to 15 min the daily scrum takes place monday morning at 9am and wednesday morning at 9am in MS Teams.
+Limited to 15 min the daily scrum takes place Monday, Tuesday and at 9 am in MS Teams.
```
Comment (WolfgangDaniel) [inline wiki/Team-Scrum-process.md:11]: *and Wednesday

**Review by WolfgangDaniel — CHANGES_REQUESTED** [review]

File: wiki/Team-Scrum-process.md (hunk 5 above)
Comment (kej-jay) [reply wiki/Team-Scrum-process.md:11]: +1

**Review by WolfgangDaniel — DISMISSED** [review]

**Review by kej-jay — DISMISSED** [review]

**Review by WolfgangDaniel — APPROVED** [review]

**Review by lukashueller — APPROVED** [review]
